packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { CliCommandExecutor } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  ContinueResponse,
+  DirFileNameSelection
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as path from 'path';
+import { Observable } from 'rxjs/Observable';
+import * as vscode from 'vscode';
+import { SfdxCommandletExecutor } from '..';
+import { channelService } from '../../channels';
+import { notificationService, ProgressNotification } from '../../notifications';
+import { taskViewService } from '../../statuses';
+import { getRootWorkspacePath, hasRootWorkspace } from '../../util';
+
+export abstract class BaseTemplateCommand extends SfdxCommandletExecutor<
+  DirFileNameSelection
+> {
+  public execute(response: ContinueResponse<DirFileNameSelection>): void {
+    const startTime = process.hrtime();
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
+    const cancellationToken = cancellationTokenSource.token;
+
+    const execution = new CliCommandExecutor(this.build(response.data), {
+      cwd: getRootWorkspacePath()
+    }).execute(cancellationToken);
+
+    execution.processExitSubject.subscribe(async data => {
+      this.logMetric(execution.command.logName, startTime);
```

Comment:
It looks like the logging for the directory type (default or custom) is missing. I think it had to do with me making this branch before I added that in 😛 . Check out one of the command files in a more recent version of develop.

Reply:
Good catch!